### PR TITLE
add review model tasks

### LINKED ISSUE #216: Azure: add review model celery tasks to run from hub
It should be all review tasks which hub run against provider + evaluate, since it use for review model process
see 
https://github.com/deeplearninc/auger-ml/blob/experimental/auger_ml/tasks_queue/tasks.py

### PR DESCRIPTION
**Pull request by skatedplrn**
Fixes #216

### CODE REVIEW

File: a2ml/tasks_queue/tasks_hub_api.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        #     logging.info("evaluate_start_task augerInfo: %s"%params['augerInfo'])
+
+    # FSClient().waitForFSReady()
+    # params = AugerML.update_task_params(params, create_project_run=False)
+    # #params['evaluate_task_id'] = evaluate_task_id
+
+    # aml = AugerML(params)
+    # aml.update_options_by_project_settings()
+    # return aml.evaluate_start(run_top_pipelines=True,
+    #                           clean_project_runs=params.get('clean_project_runs', False))
```
Comment (knobotz) [inline a2ml/tasks_queue/tasks_hub_api.py:136]: Remove all dead / commented code here.

File: tests/tasks_queue/test_tasks_api_auger.py [hunk 2]
```diff
@@ -109,19 +109,19 @@ def test_predict_success(self):
...
+        # assert len(files) == 1
 
-        assert re.match(
-            '\/auger\/models\/555777999\/predictions\/\d{4}-\d{2}-\d{2}_[0-9a-f\-]+_results.pkl.gz',
-            files[0]
-        )
+        # assert re.match(
+        #     '\/auger\/models\/555777999\/predictions\/\d{4}-\d{2}-\d{2}_[0-9a-f\-]+_results.pkl.gz',
+        #     files[0]
+        # )
```
Comment (knobotz) [inline tests/tasks_queue/test_tasks_api_auger.py:119]: Remove dead code

File: tests/tasks_queue/test_tasks_api_auger.py [hunk 3]
```diff
@@ -109,19 +109,19 @@ def test_predict_success(self):
...
 
     @vcr.use_cassette('auger/predict/invalid_model.yaml')
     def test_predict_failure_model_status(self):
-        review_bucket = self.review_bucket()
-        self.clear_bucket(review_bucket)
+        # review_bucket = self.review_bucket()
+        # self.clear_bucket(review_bucket)
```
Comment (knobotz) [inline tests/tasks_queue/test_tasks_api_auger.py:124]: Remove dead code

File: tests/tasks_queue/test_tasks_hub_api.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                        'name': 'cpucluster',
+                        'min_nodes': 0,
+                        'max_nodes': 2,
+                        'type': 'STANDARD_D2_V2'
+                    }
+                }
+            }
+        }
+        res = evaluate_start_task(params)
+        print(res)
```
Comment (knobotz) [inline tests/tasks_queue/test_tasks_hub_api.py:48]: This needs some asserts, remove print

File: tests/tasks_queue/test_tasks_api_auger.py (hunk 2 above)
Comment (holyketzer) [reply tests/tasks_queue/test_tasks_api_auger.py:119]: But without asserts this test doesn't make sense then

Comment (knobotz) [reply tests/tasks_queue/test_tasks_api_auger.py:119]: Then they should be uncommented? Doesn't make sense for them to be commented either....

**Pull request merged**